src/routing/routing_heuristics.jl

```diff
@@ -271,10 +271,10 @@ Updated mothership and tender solution with optimized waypoint positions and reg
 function optimize_waypoints(
     soln::MSTSolution,
     problem::Problem;
-    opt_method=GradientDescent(),
+    opt_method=SAMIN(ns=1), # Fminbox(GradientDescent()),
```

Comment:
Why set `ns=1`? 

This will lead to the opposite behaviour you were arguing for earlier regarding relaxing the penalty applied based on the distance to boundary